game-core/src/main/java/games/strategy/triplea/delegate/battle/firing/group/TargetGroup.java

```diff
@@ -1,4 +1,4 @@
-package games.strategy.triplea.delegate.battle;
+package games.strategy.triplea.delegate.battle.firing.group;
```

Comment:
This class was only being used by RegularFiringGroup so I moved it into the firing.group package and made the methods be package visible instead of public.